doc: Move Getting started with nRF52 Series to NCS from infocenter (#8338)
nrfconnect/sdk-nrf: doc/nrf/ug_nrf52_gs.rst

.. _ug_nrf52_gs:

Getting started with nRF52 Series
#################################

.. contents::
   :local:
   :depth: 2

This section will get you started with your nRF52 Series :term:`Development Kit (DK)` using the |NCS|.

> You will install the :ref:`peripheral_uart` sample and perform a quick test of your DK.

Author: Used "It tells you how to..." as a simples alternative.

.. _ug_nrf52_gs:

Getting started with nRF52 Series
#################################

.. contents::
   :local:
   :depth: 2

This section gets you started with your nRF52 Series :term:`Development Kit (DK)` using the |NCS|.
It tells you how to install the :ref:`peripheral_uart` sample and perform a quick test of your DK.

If you have already set up your nRF52 Series DK and want to learn more, see the following documentation:

* :ref:`ug_nrf52` for more advanced topics related to the nRF52 Series if you are already familiar with the |NCS|.
* The :ref:`introductory documentation <getting_started>` for more information on the |NCS| and the development environment.

.. _nrf52_gs_requirements:

Minimum requirements
********************

Make sure you have all the required hardware and that your computer and mobile device both have one of the supported operating systems.

Hardware
========

* One of the following nRF52 Series development kits:

  * nRF52840 DK
  * nRF52833 DK
  * nRF52 DK

* Micro-USB 2.0 cable

Software
========

On your computer, one of the following operating systems:

* Microsoft Windows 10

> * macOS X, latest version

Author: This is the way it is expressed in the other Getting started sections, though it might be changing now that we've added more information in the requirements section of the documentation.

If you have already set up your nRF52 Series DK and want to learn more, see the following documentation:

* :ref:`ug_nrf52` for more advanced topics related to the nRF52 Series if you are already familiar with the |NCS|.
* The :ref:`introductory documentation <getting_started>` for more information on the |NCS| and the development environment.

.. _nrf52_gs_requirements:

Minimum requirements
********************

Make sure you have all the required hardware and that your computer and mobile device both have one of the supported operating systems.

Hardware
========

* One of the following nRF52 Series development kits:

  * nRF52840 DK
  * nRF52833 DK
  * nRF52 DK

* Micro-USB 2.0 cable

Software
========

On your computer, one of the following operating systems:

* Microsoft Windows 10
* macOS X, latest version
* Ubuntu Linux, latest Long Term Support (LTS) version

On your mobile device, one of the following operating systems:

* Android
* iOS

.. _nrf52_gs_installing_software:

Installing the required software
********************************

On your computer, install `nRF Connect for Desktop`_.

> After installing and starting the application, install the Programmer app.

Reviewer: ```suggestion
After installing and starting the application, install the Programmer application.
```

Reviewer: Or is just referred to as "Programmer app"? I see this on line 81 as well.

Author: I believe it's generally considered an app since it's inside the nRF Connect for Desktop application. At least, that's how I've referred to it before.

You must also install a terminal emulator, such as :ref:`PuTTy <putty>` or the `nRF Terminal <nRF Terminal documentation_>`_ extension which is part of the |nRFVSC|.
The steps detailed in :ref:`nrf52_gs_connecting` use PuTTy, but any terminal emulator will work.

On your mobile device, install the `nRF Connect for Mobile`_ mobile application from the corresponding application store.

.. _nrf52_gs_installing_application:

Installing the application
**************************

You must program and run a precompiled version of the :ref:`peripheral_uart` sample application on your development kit to test the functions.

Download the precompiled version of the application for your DK from the corresponding download page:

* `nRF52840 DK Downloads`_
* `nRF52833 DK Downloads`_
* `nRF52 DK Downloads`_

After downloading the zip archive, extract it to a folder of your choice.
The archive contains the HEX file used to program the application to your DK.

To program the precompiled sample application to your development kit, complete the following steps:

> 1. Open the Programmer app.

Author: I think it's fine since we do instruct the user to install it earlier on the page.

> #. Connect the nRF52 Series DK to the computer with a micro-USB cable, and then turn on the DK.

Author: Fixed.

   **LED1** starts blinking.

#. Click **SELECT DEVICE** and select the DK from the drop-down list.

   .. figure:: /images/programmer_select_device1.png
      :alt: Programmer - Select Device

      Programmer - Select Device

   The drop-down text changes to the type of the selected device, with its SEGGER ID below the name.
   The **Device Memory Layout** section also changes its name to the device name, and indicates that the device is connected.
   If the **Auto read memory** option is selected in the **DEVICE** section of the side panel, the memory layout will update.
   If it is not selected and you wish to see the memory layout, click :guilabel:`Read` in the **DEVICE** section of the side panel.

#. Click :guilabel:`Add file` in the **FILE** section, and select **Browse**.
#. Navigate to where you extracted the application HEX file and select it.

> #. Click the :guilabel:`Erase & write` button in the **DEVICE** section to program the DK.
>    Do not unplug or turn off the DK during this process.

Reviewer 2: ```suggestion
#. Click the :guilabel:`Erase & write` button in the **DEVICE** section to program the DK.

   Do not unplug or turn off the DK during this process.
```

Author: Fixed.

.. note::

>    If you experience any problems during the process, restart the Programmer app by pressing Ctrl+R (command+R on macOS), and try again.

Reviewer 2: ```suggestion
   If you experience any problems during the process, press Ctrl+R (command+R on macOS) to restart the Programmer app, and try again.
```

Author: Fixed.

> After you have programmed the application to the DK, you can connect to it, and then test the functions.

Reviewer 2: ```suggestion
After you have programmed the application to the DK, you can connect to it and test the functions.
```

Author: Fixed.

   #. Scroll down and expand **Ports (COM & LPT)**.
   #. Find the port named *JLink CDC UART Port* and note down the number in parentheses.

      If you have more than one J-Link UART Port, unplug the one that you want to use, plug it back in, and observe which one appeared last.

      Your DK can also show up as two consecutive COM ports.
      If this is the case, you need to test which COM port is the correct one.

#. Configure the following settings:

   * Speed (baud): 115200
   * Data bits: 8
   * Stop bits: 1
   * Parity: None
   * Flow control: RTS/CTS

#. Click :guilabel:`Open` to open the terminal window.

The connection has now been established.
If you test the application now, you can go directly to Step 2 of :ref:`nrf52_gs_testing`.

.. _nrf52_gs_testing:

Testing the application
***********************

You can test the :ref:`peripheral_uart` application on your DK using the `nRF Connect for Mobile`_ mobile application.
The test requires that you have :ref:`connected to the application <nrf52_gs_connecting>` and have the connected terminal emulator open.

To perform tests, complete the following steps:

.. tabs::

   .. group-tab:: Android

>       1. Connect the nRF52 Series DK to the computer with a micro-USB cable, and then turn on the DK.

Reviewer 2: ```suggestion
      1. Connect the nRF52 Series DK to the computer with a micro-USB cable and turn on the DK.
```

Author: Fixed.

      #. In nRF Connect for Mobile, tap :guilabel:`Scan`.
      #. Find the DK in the list, select it and tap :guilabel:`Connect`.

         The default device name for the Peripheral UART sample is *Nordic_UART_Service*.

      #. When connected, tap the three-dot menu below the device name, and select **Enable CCCDs**.

         This example communicates over Bluetooth Low Energy using the Nordic UART Service (NUS).

         .. figure:: /images/nrf52_enable_cccds.png
            :alt: nRF Connect for Mobile - Enable services option

            nRF Connect for Mobile - Enable services option

      #. Tap the three-dot menu next to **Disconnect** and select **Show log**.
      #. On your computer, in the terminal emulator connected to the application through CDC-UART, type ``hello`` and send it to the DK.

         The text is sent through the nRF52 Series DK to your mobile device over a Bluetooth LE link.
         The device displays the text in the nRF Connect for Mobile log:

         .. figure:: /images/nrf52_connect_log.png
            :alt: nRF Connect for Mobile - Text shown in the log

            nRF Connect for Mobile - Text shown in the log

   .. group-tab:: iOS

>       1. Connect the nRF52 Series DK to the computer with a micro-USB cable, and then turn on the DK.

Reviewer 2: ```suggestion
      1. Connect the nRF52 Series DK to the computer with a micro-USB cable and turn on the DK.
```

Author: Fixed.
